godtools/App/Share/SwiftUI Views/ProgressBarView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ProgressBarView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 5/5/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct ProgressBarView: View {
```

Comment:
Oh weird!  What iOS version were you testing on?  I was seeing it animate starting from the left on 15.4

Reply:
Hey @rachaelblue I was in iPhon 8 13.4 simulator.  I wonder if it has to do with an alignment?  Maybe 15.4 changed something with the alignment?  Hopefully that is it and nothing too complicated.

Reply:
Yeah probably just need to specify alignment.  Will check it out on 13.4.  Thanks!

Reply:
I think this is a bug with iOS 13, because it doesn't happen on 14/15, and setting any extra alignment properties anywhere doesn't fix it.  So I added a spacer, which shouldn't really be necessary, but it worked.

<img width="569" alt="Screen Shot 2022-05-06 at 2 40 12 PM" src="https://user-images.githubusercontent.com/8647064/167199375-f8f62e04-45f4-487f-80f4-93606139d1e2.png">

Reply:
Thanks I'll take a look.  SwiftUI in iOS 13 must have some differences between 14 and 15.  Makes me wonder what was changed.